src/tests/unit/services/GenerateVerifiableCredential.test.ts

```diff
@@ -231,6 +254,34 @@ describe("GenerateVerifiableCredential", () => {
 		);
 	});
 
+	it.each([
```

Comment:
This doesn't need to be an each if there is only one scenario, lets change it to a regular `it`